fastlane/lib/fastlane/plugins/template/.rubocop.yml

```diff
@@ -0,0 +1 @@
+../../../../../.rubocop.yml
```

Comment:
Alright, I reduced the size of the `.rubocop.yml`, please copy the file. It might be that it's not bundled with the actual gem, so you might need to update `fastlane.gemspec` (which might not allow the use of `../.rubocop.yml` as a valid path)